database/migrations/create_nfl_kick_stats_table.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('nfl_kick_stats', function (Blueprint $table) {
+            $table->id();
+            $table->integer('season');
+            $table->integer('week');
+            $table->string('season_type')->nullable();
+            $table->string('team');
+            $table->string('player_name')->nullable();
+            $table->string('player_id');
```

Comment:
Is it possible for player_id to be bull?

Reply:
no.